hr_payslip_change_state/README.rst

```diff
@@ -0,0 +1,53 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=========================
+HR Payslip Change State
+=========================
+
+This module process payslip batches:
+
+* This module allows to execute the actions that buttons inside a form do \
+  for a payslip batch.
```

Comment:
Do you mean "This module allows to mass change Payslip states"?
That's the feature provided, correct?